tests/test_metadata.py

```diff
@@ -4,16 +4,36 @@
 from nose.tools import *  # PEP8 asserts
 
 from framework.forms.utils import process_payload
+from framework.mongo import database as database_proxy, set_up_storage
+from modularodm import storage
+from modularodm.exceptions import KeyExistsException
 
+import website.models
+from website import settings
 from website.project.model import MetaSchema
 from website.project.model import ensure_schemas
 from website.project.metadata.schemas import OSF_META_SCHEMAS
 
-from tests.base import OsfTestCase
+from tests.base import OsfTestCase, teardown_database
 
 
 class TestMetaData(OsfTestCase):
 
+    def tearDown(self):
+        super(TestMetaData, self).tearDown()
+
+        # For whatever reason, DbTestCase only wipes the database during
```

Comment:
>For whatever reason

Probably efficiency, but I don't know that anyone has timed it to see if it makes a significant difference.